src/app/courses/courses.component.ts

```diff
@@ -1,28 +1,31 @@
-import { Component, OnInit } from '@angular/core';
-import { Courses, CoursesService } from '../shared/services/courses.service';
-import { ActivatedRoute, Router, NavigationEnd } from '@angular/router';
-import { SubscriptionsManager } from '../shared/base/subscriptions-manager';
-
+import { Component, OnInit } from "@angular/core";
+import { Courses, CoursesService } from "../shared/services/courses.service";
+import { ActivatedRoute, Router, NavigationEnd } from "@angular/router";
+import { SubscriptionsManager } from "../shared/base/subscriptions-manager";
 
 @Component({
-  selector: 'app-courses',
-  templateUrl: './courses.component.html',
-  styleUrls: ['./courses.component.scss']
+  selector: "app-courses",
```

Comment:
@Yuriy-27 don't worry about " or ', just define which you prefer more and use one type in .ts files.